cmd/kubeadm/app/phases/upgrade/compute.go

```diff
@@ -23,7 +23,7 @@ import (
 	kubeadmconstants "k8s.io/kubernetes/cmd/kubeadm/app/constants"
 	"k8s.io/kubernetes/cmd/kubeadm/app/features"
 	"k8s.io/kubernetes/cmd/kubeadm/app/phases/addons/dns"
-	"k8s.io/kubernetes/cmd/kubeadm/app/util"
+	etcdutil "k8s.io/kubernetes/cmd/kubeadm/app/util/etcd"
```

Comment:
I neglected to put this in the commit, but it was necessary for me to restructure the module, because `HasTLS()` imports `ReadStaticPodFromDisk ` from `app/util/staticpod`

`app/util/staticpod` already imports from `/app/util` and this created a dependency cycle.